copulas/multivariate/gaussian.py

```diff
@@ -70,7 +71,11 @@ def _get_covariance(self, X):
 
         """
         result = self._transform_to_normal(X)
-        return pd.DataFrame(data=result).cov().values
+        covariance = pd.DataFrame(data=result).cov().values
```

Comment:
This adds noise to the diagonal if the matrix has a large condition number, allowing us to avoid issues with singular matrices in the `pdf`/`cdf`. Alternatively, we could just check for constant columns and remove them before modeling.